lib/providers/qwen.sh

```
@@ -0,0 +1,108 @@
+  exit 1
+fi
+
+# ── Build command ─────────────────────────────────────────────────
+QWEN_ARGS=(
+  -p
+  --yolo
+)
+
+if [[ -n "$PROMPT_FILE" ]]; then
+  # Guard against very large prompt files (ARG_MAX limit)
+  local prompt_size
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)